jax/core.py

```diff
@@ -462,6 +464,10 @@ def __len__(self):
   def aval(self):
     raise NotImplementedError("must override")
 
+  @property
+  def _live(self):
```

Comment:
Do you expect other kinds of tracers to want to implement this as returning `True` or `False`? Otherwise, the implementation you're using here (True or error) is maybe a better fit for an `_assert_live(self)` method that errors or returns `None`.

Reply:
Good point! I think True or error is better. I'll try to tweak it.